sigs.yaml

```diff
@@ -1,38 +1,27 @@
 sigs:
+    - dir: sig-api
+      name: api
+      label: sig/api
+    - dir: sig-observability
+      name: observability
+      label: sig/observability
+    - dir: sig-release
+      name: release
+      label: sig/release
+      subprojects:
+        - name: kubevirt
+          owners:
+            - https://raw.githubusercontent.com/kubevirt/sig-release/main/OWNERS
     - dir: sig-compute
       name: compute
+      label: sig/compute
       subprojects:
         - name: kubevirt
           owners:
-            - https://raw.githubusercontent.com/kubevirt/community/main/OWNERS
```

Comment:
Background on this: we are generating associations for sig activities inside [kubevirt devstats](https://kubevirt.devstats.cncf.io/) with these entries.

See [kubevirt/community](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/community/community-postsubmits.yaml#L3) postsubmit for more details

Reply:
ACK understood but I think we need to look at moving lots of these deprecated/stale projects elsewhere.

Reply:
Or maybe we should contact project maintainers and ask them whether it would be safe to archive them? 

Note: Archiving can be done over kubevirt/project-infra orgs.yaml

Reply:
Reverted, yeah we really should but I'm not going to tackle that as part of this effort.

Reply:
Created https://github.com/kubevirt/community/issues/239 as a follow up.

Reply:
Okay I lied, I'll look at this for some of the obvious projects under https://github.com/kubevirt/project-infra/pull/2942